### ISSUE TITLE
[Feature Request] When you click the watch later button or when you put a video into a playlist it gets marked as watched by the extension.

### ISSUE THREAD

**Issue by NotSoHeroic** [OsaSoft/youtube-better-subscriptions#186]
Hi, just started using the extension very recently, and I use a lot of similar extensions and I like being able to mark videos as watched with this extension after I've already sorted where I want a video to go, usually by just clicking the watch later button to filter the majority of the videos into that built in playlist. But for others I have playlists to sort them into.

I would find it convenient if there was a setting that when ticked on makes it so that once you've put a video into any playlist, including by clicking the "Watch Later" button in the top right of the thumbnail or when manually clicking the hamburger and selecting "Save to playlist" and then selecting your playlist(s), I'd like (if checked in the settings) for this extension to automatically hide those videos once those actions are taken. 

This would essentially just eliminate the extra button click but it would save a lot of time when sorting through my subscription page.

I know my setup most likely looks different than a lot of other users since I have tons of extensions installed for YouTube for minimal changes, but a setting like this would significantly speed things up, and I'm just realizing while typing this that I could use the "Mark all as watched" button after I'm done sorting through all the videos, but that doesn't exactly achieve the same thing as I don't necessarily want every video in the extension's watched list.

But asking for a feature that scans all your playlists and removes all the videos in those playlists from the subscription page to the extensions watched list seems like too big of a task for something that could just be an quick fix and auto add the videos to the extensions watched list when you click the buttons that already exist on YouTube. Like the Watch Later button; or saving to a playlist; or even adding to queue; or any of the other buttons in the drop down list.

I really enjoy this feature because it makes it so that if you click off the subscription page or exit out you can come back to the exact point you were far more quickly then without the hide videos feature. Sometimes if I fall a few days behind it can be a chore to have to scroll through until getting to an older date, with this extension feature I can just have most of the videos already in my playlists hidden and can get to the ones that aren't much quicker, it's night and day. But implementing the feature I've asked for would be the icing on the cake, I'd be able to speed through and sort these specific videos much more quickly on a daily basis.

Hope that all made sense.

**Comment by tummyslyunopened** [comment]
TL;DR in good faith:
[Feature Request] Automatically mark videos as "watched" (and hide them) when they're added to any playlist (like "Watch Later" or custom ones), without needing to manually click "Mark as watched." This would save time when organizing videos, especially on the subscription page. They note that this would speed up their workflow and make the extension even more useful.

Suggested approach:
Querying playlist entries to automatically remove these seems out of scope for this project, but in latest UI on my machine:

This Selector Corresponds to the [3-dot] icon that is clicked to add video to playlist from subs feed. 

```css
'#button > yt-icon > span > div'
```

<img width="711" height="324" alt="Image" src="https://github.com/user-attachments/assets/07d380fb-41e2-4bec-ac0c-0c4c142c789d" />


which spawns separate element for playlist addition selection.

<img width="771" height="775" alt="Image" src="https://github.com/user-attachments/assets/d262c5d9-1e2a-4d15-8ba2-b71156bc58bc" />

Since it is not nested within the video preview container, it will present the playlist selection and watch later even if the thumbnail is already hidded. So you will have a chance to select playlist etc, but the preview would already be hidden. 

So I propose option default false:

<img width="536" height="65" alt="Image" src="https://github.com/user-attachments/assets/9f766884-baf9-4eb5-ae56-86220abc23ca" />
